suite-native/video-assets/src/components/Video.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { View, ActivityIndicator } from 'react-native';
+
+import { Video as ExpoVideo, VideoProps as ExpoVideoProps } from 'expo-av';
+
+import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
+
+import { videos, VideoName } from '../videos';
+
+type VideoProps = ExpoVideoProps & {
+    name: VideoName;
+    aspectRatio?: number;
+};
+
+type VideoStyleProps = {
+    aspectRatio: number;
+};
+
+const videoContainer = prepareNativeStyle((_, { aspectRatio }: VideoStyleProps) => ({
+    flex: 1,
+    aspectRatio,
```

Comment:
I have bad experience with `aspectRatio` what about just `width: 100%; heigh: auto`? It should work fine and we don't need to calculate it.

Reply:
What is the problem with using `aspectRatio`? When I try your proposed solution, `height: 'auto'` sets the height to `0` and the video is hidden.